x-pack/plugins/security_solution/public/common/components/markdown_editor/plugins/timeline/parser.ts

```diff
@@ -5,7 +5,7 @@
  * 2.0.
  */
 
-import { Plugin } from '@elastic/eui/node_modules/unified';
+import { Plugin } from 'unified';
```

Comment:
This code expects the `unified` package to be both a sub-dependency of `@elastic/eui` (which there's no guarantee of) + that it would conflict with another `unified` package and be placed in it's own `node_modules` folder under `./node_modules/@elastic/eui/node_modules`.

This PR consolidated our different versions of `unified` and as a result, it no longer had to live under `./node_modules/@elastic/eui/node_modules`, which meant that this code broke.

In general one should _never_ expect a package to exist in a specific location on disk. The yarn/npm package mangers make no guarantees about this.

Second, all dependencies used by our code should be listed in our `package.json` file as either a dependency or a dev-dependency. This PR also adds it as a `dependency`.

Reply:
Checked and we do not use Q anywhere else. Thanks for this. I think our usage of Q was accidental and not really what we want.